commons-vfs2-jackrabbit1/pom.xml

```diff
@@ -111,6 +111,9 @@
 
   <properties>
     <vfs.parent.dir>${basedir}/..</vfs.parent.dir>
+    <commons.componentid>vfs-jackrabbit1</commons.componentid>
+    <commons.packageId>vfs-jackrabbit1</commons.packageId>
```

Comment:
@satish-csi 
Thank you for your PR.
That does not seem right since all package IDs in Apache Commons are/should-be prefixed with "commons-". 
What about overriding the OSGi ID?

Reply:
@garydgregory This is parent commons pom of VFS master pom
https://repo1.maven.org/maven2/org/apache/commons/commons-parent/50/commons-parent-50.pom

`<commons.osgi.symbolicName>org.apache.commons.${commons.packageId}</commons.osgi.symbolicName>`
it does include "commons"
Here is symbolic name generated by jackrabbit2 module with above settings
**Bundle-SymbolicName: org.apache.commons.vfs-jackrabbit2**

Reply:
@garydgregory corrected both names to
```
<commons.componentid>vfs2-jackrabbit1</commons.componentid>
<commons.packageId>vfs2-jackrabbit1</commons.packageId>
```

Reply:
Still not right IMO, see my previous comment.

Reply:
@garydgregory current master pom package id is
**`<commons.packageId>vfs2</commons.packageId>`**
I tried to make it on similar pattern, Could you please clue me what more thing I need to change?

Reply:
below are properties are defined in commons pom which one we need to override?
```
<commons.packageId>${project.artifactId}</commons.packageId>
<!--  Configuration properties for the OSGi maven-bundle-plugin  -->
<commons.osgi.symbolicName>org.apache.commons.${commons.packageId}</commons.osgi.symbolicName>
<commons.osgi.export>org.apache.commons.*;version=${project.version};-noimport:=true</commons.osgi.export>
<commons.osgi.import>*</commons.osgi.import>
<commons.osgi.dynamicImport/>
<commons.osgi.private/>
<commons.osgi.excludeDependencies>true</commons.osgi.excludeDependencies>
```

Reply:
@garydgregory package ID for main project is
`**<commons.packageId>vfs2</commons.packageId>**`

Reply:
Oops I though we were using the commons.packsgeId as an artifact id.